core/src/com/unciv/models/ruleset/unique/UniqueParameterType.kt

```diff
@@ -262,6 +262,7 @@ enum class UniqueParameterType(
             "in annexed cities", "Annexed",
             "in puppeted cities", "Puppeted",
             "in resisting cities", "Resisting",
+            "in founded cities", "Founded",
```

Comment:
I don't like this phrasing. What is a 'founded' city? All cities are founded 
I don't have a better suggestion but this doesn't pass, in my opinion